helper_funcs.inc.php

```diff
@@ -210,6 +210,11 @@ function pwg_privacy_verify_access ($img_id, $req_path) {
 
 	//file is original uploaded image
 	if ( strpos($req_path, $path) === 0 ) {
+		//file is a video, serve it.
+		if ( strpos($path, '.mp4') !== 0 ) {
```

Comment:
strpos returns `false` if it can't find the needle in the haystack.
0 is returned only if it is the first match like '.mp4file'